### ISSUE TITLE
No consistent way to use loose list or tight list

### ISSUE THREAD

**Issue by qiulang** [typora/typora-issues#3467]
When I use order list I found typora seemed not to have a consistent way to use loose list or tight list. Sometime it will use loose list, e.g. with extra blank line, but sometime uses tight list.

Is this a bug or I just haven't figured out the way to trigger typora to use loop list or tight list ?

Thanks

**Comment by abnerlee** [comment]
Press "Return" key will make a new paragraph. When there are paragraphs in markdown, there will be empty lines, then the list become "loose".

So for tight list, if it contains more than one line in a list, you can use "Shift+Return" to make line break, which makes a new line rather than a paragraph.

Or switch to source code mode to remove empty lines.

**Comment by qiulang** [comment]
What you said can not explain what I experienced. My question is I can't find a consistent way to use loose list or tight list. Let me give you can example. 

The text in Chinese but you should be easy to see my point.

<img width="1237" alt="t1" src="https://user-images.githubusercontent.com/516243/83715957-3174d400-a661-11ea-9904-ffcf1ed5addd.png">

Then in the next commit I have no idea why Typora changed it to loose list. I didn't do what you said to change it from tight list to loose list. So I don't understand why Typora changed it

<img width="1193" alt="t2" src="https://user-images.githubusercontent.com/516243/83716185-c546a000-a661-11ea-8987-6810d8b69d49.png">

**Comment by abnerlee** [comment]
Hi

What's your Typora version? and which operations did you make from pic 1 to pic 2?

**Comment by qiulang** [comment]
I use the latest version **0.9.9.33.2 (4442)** but I believe it exists in all versions I have used.

I didn't remember what I did to trigger typora to change it loose list to tight list, but I am sure I didn't use "Shift+Return" to make line break.

Actually I didn't understand why typora change tight list to loose list and that was the reason I opened this issue in the first place! I just know it happens from time to time. I will try to find a way to reproduce the problem!

**Comment by qiulang** [comment]
Hi I hit this problem again and I attached to a md file to trigger it.

So in commit df8fd91, tll4.md is tight list

<img width="1306" alt="Screen Shot 2020-08-17 at 11 52 17 AM" src="https://user-images.githubusercontent.com/516243/90355828-1b7f8880-e080-11ea-935c-429bf7dd01dc.png">

While in commit 59f56a0, typora changed it loose list

What I did is at the end of line 14, I hit enter twice and typora changed it to loose list.

But it is strange that when I want to reproduce the problem, restored the file the tight list and do it again, typora won't change it to loose list. Only the first time!



<img width="1315" alt="Screen Shot 2020-08-17 at 11 54 17 AM" src="https://user-images.githubusercontent.com/516243/90355923-63061480-e080-11ea-9423-83ebfb8e0ece.png">
